devops/helm/redis-cluster/values.yaml

```diff
@@ -0,0 +1,1202 @@
+    ##
+    enabled: false
+    ## @param cluster.externalAccess.hostMode Set cluster preferred endpoint type as hostname
+    ## ref: https://github.com/redis/redis/pull/9530
+    ##
+    hostMode: false
+    service:
+      ## @param cluster.externalAccess.service.disableLoadBalancerIP Disable use of `Service.spec.loadBalancerIP`
+      ##
+      disableLoadBalancerIP: false
+      ## @param cluster.externalAccess.service.loadBalancerIPAnnotaion Name of annotation to specify fixed IP for service in. Disables `Service.spec.loadBalancerIP` if not empty
+      ##
+      loadBalancerIPAnnotaion: ""
+      ## @param cluster.externalAccess.service.type Type for the services used to expose every Pod
+      ## At this moment only LoadBalancer is supported
+      ##
+      type: LoadBalancer
+      ## @param cluster.externalAccess.service.port Port for the services used to expose every Pod
+      ##
+      port: 6379
+      ## @param cluster.externalAccess.service.loadBalancerIP Array of load balancer IPs for each Redis&reg; node. Length must be the same as cluster.nodes
+      ##
+      loadBalancerIP: []
+      ## @param cluster.externalAccess.service.loadBalancerClass Load Balancer class if service type is `LoadBalancer` (optional, cloud specific)
+      ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-loadbalancer
+      ##
+      loadBalancerClass: ""
+      ## @param cluster.externalAccess.service.loadBalancerSourceRanges Service Load Balancer sources
+      ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+      ## e.g:
+      ## loadBalancerSourceRanges:
+      ##   - 10.10.10.0/24
+      ##
+      loadBalancerSourceRanges: []
+      ## @param cluster.externalAccess.service.annotations Annotations to add to the services used to expose every Pod of the Redis&reg; Cluster
+      ##
+      annotations: {}
+  ## This section allows to update the Redis&reg; cluster nodes.
+  ##
+  update:
+    ## @param cluster.update.addNodes Boolean to specify if you want to add nodes after the upgrade
+    ## Setting this to true a hook will add nodes to the Redis&reg; cluster after the upgrade. currentNumberOfNodes and currentNumberOfReplicas is required
+    ##
+    addNodes: false
+    ## @param cluster.update.currentNumberOfNodes Number of currently deployed Redis&reg; nodes
+    ##
+    currentNumberOfNodes: 6
+    ## @param cluster.update.currentNumberOfReplicas Number of currently deployed Redis&reg; replicas
+    ##
+    currentNumberOfReplicas: 1
+    ## @param cluster.update.newExternalIPs External IPs obtained from the services for the new nodes to add to the cluster
+    ##
+    newExternalIPs: []
+## @section Metrics sidecar parameters
+##
+
+## Prometheus Exporter / Metrics
+##
+metrics:
+  ## @param metrics.enabled Start a side-car prometheus exporter
+  ##
+  enabled: false
```

Comment:
If time allows, can we have the metrics enabled to check the monitoring in sysdig?

Reply:
Will verify in the future PR